es6-classes/circle.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Circle */
+class Circle extends Shape {
+  constructor(radius) {
+    const area = Math.PI * radius * radius;
+    const perimeter = 2 * Math.PI * radius;
```

Comment:
Rather than creating `area` & `perimeter` properties directly, you should be utilizing the `super` constructor to create them for you